lumigator/backend/backend/services/workflows.py

```diff
@@ -161,19 +190,12 @@ async def _run_inference_eval_pipeline(
             # we'll make that improvement later
             # Get the dataset from the S3 bucket
             result_key = self._job_service._get_results_s3_key(evaluation_job.id)
+            loguru.logger.info(f"METRICS {eval_output.metrics}")
+
+            formatted_metrics = self._prepare_metrics(eval_output)
 
             outputs = RunOutputs(
-                metrics={
-                    "rouge1_mean": round(eval_output.metrics["rouge"]["rouge1_mean"], 3),
-                    "rouge2_mean": round(eval_output.metrics["rouge"]["rouge2_mean"], 3),
-                    "rougeL_mean": round(eval_output.metrics["rouge"]["rougeL_mean"], 3),
-                    "rougeLsum_mean": round(eval_output.metrics["rouge"]["rougeLsum_mean"], 3),
-                    "bertscore_f1_mean": round(eval_output.metrics["bertscore"]["f1_mean"], 3),
-                    "bertscore_precision_mean": round(eval_output.metrics["bertscore"]["precision_mean"], 3),
-                    "bertscore_recall_mean": round(eval_output.metrics["bertscore"]["recall_mean"], 3),
-                    "meteor_mean": round(eval_output.metrics["meteor"]["meteor_mean"], 3),
```

Comment:
@khaledosman Is this change going to affect the frontend? If you access information via `workflow.rouge1_mean` it would now be at `workflow.rouge_rouge1_mean` since I now preprocess to flatten the array instead of manually specifying metrics like is happening here.

I see the "Metrics" type in `lumigator/frontend/src/types/Workflow.ts` which looks like it might impact, not sure. I can update that file in this PR if my understanding is correct.

Reply:
Ok, it's all ok because `lumigator/frontend/src/helpers/transformJobResults.ts` `transformJobResults` is taking the mean from the job, not the workflow, so the frontend doesn't need to be updated to be compatible with this change 👍